TeknoParrotUi.Common/GameProfiles/Theatrhythm.xml

```diff
@@ -1,14 +1,14 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <GameProfile xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
-  <GameName>Theatrhythm Final Fantasy: All-Star Carnival</GameName>
+  <GameName>Theatrhythm: Final Fantasy - All-Star Carnival</GameName>
```

Comment:
Theatrhythm Final Fantasy: All-Star Carnival is the official name.

Reply:
OK, re-issued.